server/graphql/v2/object/Host.js

```diff
@@ -437,6 +464,135 @@ export const Host = new GraphQLObjectType({
+          return {
+            contributionsCount: () =>
+              models.Transaction.count({
+                where,
+              }),
+            oneTimeContributionsCount: () =>
+              models.Transaction.count({
+                where,
+                include: [{ model: models.Order, where: { interval: null } }],
+                ...distinct,
+              }),
+            recurringContributionsCount: () =>
+              models.Transaction.count({
+                where,
+                include: [{ model: models.Order, where: { interval: { [Op.ne]: null } } }],
+                ...distinct,
+              }),
+            dailyAverageIncomeAmount: {
+              value: dailyAverageIncomeAmount,
+              currency: host.currency,
+            },
+          };
+        },
+      },
+      expenseStats: {
+        type: new GraphQLNonNull(ExpenseStats),
+        args: {
+          account: {
+            type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
+            description: 'A collection of accounts for which the expense stats should be returned.',
+          },
+          dateFrom: {
+            type: GraphQLDateTime,
+            description: 'Calculate expense statistics beginning from this date.',
+          },
+          dateTo: {
+            type: GraphQLDateTime,
+            description: 'Calculate expense statistics until this date.',
+          },
+        },
+        async resolve(host, args, req) {
+          if (!req.remoteUser?.isAdmin(host.id)) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see the expense stats.');
+          }
+          const where = { HostCollectiveId: host.id, kind: 'EXPENSE', type: TransactionTypes.DEBIT };
+          const numberOfDays = getNumberOfDays(args.dateFrom, args.dateTo, host);
+          const dateRange = getFilterDateRange(args.dateFrom, args.dateTo);
+          if (dateRange) {
+            where.createdAt = dateRange;
+          }
+          if (args.account) {
+            const collectives = await fetchAccountsWithReferences(args.account, { throwIfMissing: true });
+            const collectiveIds = collectives.map(collective => collective.id);
+            where.CollectiveId = { [Op.in]: collectiveIds };
+          }
+          const distinct = { distinct: true, col: 'ExpenseId' };
+
+          const expensesAmountSum = await models.Transaction.sum('amount', { where });
+          const dailyAverageAmount = expensesAmountSum ? Math.abs(expensesAmountSum) / numberOfDays : 0;
+          return {
+            expensesCount: () =>
+              models.Transaction.count({
+                where,
+                ...distinct,
+              }),
+            invoicesCount: models.Transaction.count({
+              where,
+              include: [{ model: models.Expense, where: { type: expenseType.INVOICE } }],
+              distinct,
```

Comment:
Here and in other places I think we're missing the `...`, we're currently passing options as `options.distinct.distinct = true`.

Reply:
My mistake; thanks for pointing it out. 👍🏽